app/src/main/java/org/greenstand/android/TreeTracker/database/dao/TreeDto.kt

```diff
@@ -10,6 +10,7 @@ class TreeDto {
     var longitude: Double = 0.0
     var accuracy: Int = 0
     var name: String? = null
+    var note: String? = null
```

Comment:
Wasn't the note previously the content value on line 15? If that's the case, it should be removed.

Reply:
Yup, I changed the name elsewhere but missed it in the model I guess.